Calculate_Mutation_Models/workflow/calculate_mutation_models_synapse_workflow.cwl

```diff
@@ -15,6 +20,7 @@ inputs:
   groups_synapse_id: string
   mutations_synapse_id: string
   output_parent_synapse_id: string
+  output_name_on_synapse: string?
```

Comment:
Could you remove this and then use output file down below.
```suggestion
```

Reply:
Good catch!